kodilla-hibernate/src/main/java/com/kodilla/hibernate/tasklist/TaskList.java

```diff
@@ -0,0 +1,48 @@
+package com.kodilla.hibernate.tasklist;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "TASKLIST")
+public class TaskList {
+
+    private int id;
+    private String listName;
+    private String description;
+
+    public TaskList() {
+    }
+
+    public TaskList(String listName, String description) {
+        this.listName = listName;
+        this.description = description;
+    }
+
+    @Id
+    @GeneratedValue
+    @Column(unique = true)
```

Comment:
Po co to unique? Jak cos jest @Id to jest unique

Reply:
Poprawione, myślałem że unique musi zostać